RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -18,6 +18,10 @@
   trackClassifier       = cms.InputTag('tobTecStep',"QualityMasks"),
 )
 
+# turn off conversions for heavy ions without removing collections                                                                  
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+pp_on_AA_2018.toModify(convClusters, maxChi2 = 0.)
```

Comment:
it looks like this change doesn't do what the comment is implying.
IIUC, this maxChi2 is used to select clusters which are clearly belonging to a previous (tobtec) iteration tracks and should not be used in the following iteration.
By changing to maxchi2=0 all hits on tobtec tracks will be allowed to be considered in the conv step tracking.
So, the effect is kind of the opposite of what the comment says and perhaps this modification should be removed.

It looks like the following would achieve some reduction
``` photonConvTrajSeedFromSingleLeg.RegionFactoryPSet.RegionPSet.ptMin = 999.```